docs/methods/cardanoGetAddress.md

```diff
@@ -1,44 +1,39 @@
 ## Cardano: get address
-Display requested address derived by given [BIP32-Ed25519](https://cardanolaunch.com/assets/Ed25519_BIP.pdf) path on device and returns it to caller. User is presented with a description of the requested key and asked to confirm the export on Trezor.
+Display requested address derived by given BIP32-Ed25519 path on device and return it to caller.
 
 ES6
 ```javascript
 const result = await TrezorConnect.cardanoGetAddress(params);
 ```
 
-CommonJS
-```javascript
-TrezorConnect.cardanoGetAddress(params).then(function(result) {
-
-});
-```
 
 ### Params
-[****Optional common params****](commonParams.md)
+
+Has 2 modes, the single key export, and the bundle export.
+
 #### Exporting single address
-* `addressParameters` — *obligatory* see description below
-* `address` — *optional* `string` address for validation (read `Handle button request` section below)
-* `protocolMagic` - *obligatory* `Integer` 764824073 for Mainnet, 42 for Testnet
-* `networkId` - *obligatory* `Integer` 1 for Mainnet, 0 for Testnet
-* `showOnTrezor` — *optional* `boolean` determines if address will be displayed on device. Default is set to `true`
+* `addressParameters` — *required* see description below
+* `address` — *optional* `string` Address in Bech32/Base58 for custom validation, read `Custom UI validation` section below
+* `protocolMagic` - *required* `number`  must be 764824073 for Mainnet, 42 for Testnet
```

Comment:
Technically, you can use any other protocol magic and Trezor would just show it as an "unknown network" so I would add, a remark that "other values are accepted as well but treated as "unknown networks""